### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by Rishi-Mishra0704**
# Setup for Math magician project

- Add MIT license
- Update readme
- Add linters

### CODE REVIEW

**Comment by prolajumokeoni** [comment]
Hi @Rishi-Mishra0704, kindly add the future features section to your readme 👍🏿

**Comment by Rishi-Mishra0704** [comment]
@prolajumokeoni I can't think of any key features as there is nothing , it is only a setup for the project

**Comment by Rishi-Mishra0704** [comment]
@prolajumokeoni I have updated the readme, is there any other issue or you can review it ?

**Comment by prolajumokeoni** [comment]
Hi @Rishi-Mishra0704 


While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have the future features section in your readme

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by prolajumokeoni** [comment]
> @prolajumokeoni I have updated the readme, is there any other issue or you can review it ?

Hi @Rishi-Mishra0704, Kindly add content to the future features section, please check the [ big project picture  ](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_setup.md#need-a-big-picture) to help you coin future features for this milestone

File: README.md [hunk 1]
```diff
@@ -1,70 +1,139 @@
...
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank everyone contributed the completion of this project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
```
Comment (Davidosky007) [inline README.md]: * Kindly fix your MIT link its broken 🙏

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @Rishi-Mishra0704 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* No linter errors 👍 
* Descriptive PR 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by prolajumokeoni — APPROVED** [review]
Hi @Rishi-Mishra0704 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- All changes requested have been implemented   ✅ 
  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**